iOS/fastlane/Matchfile

```diff
@@ -16,35 +16,29 @@ for_lane :sync_signing_alpha_adhoc do
     type "adhoc"
     app_identifier ["com.duckduckgo.mobile.ios.alpha", "com.duckduckgo.mobile.ios.alpha.ShareExtension", "com.duckduckgo.mobile.ios.alpha.OpenAction2", "com.duckduckgo.mobile.ios.alpha.Widgets", "com.duckduckgo.mobile.ios.alpha.NetworkExtension", "com.duckduckgo.mobile.ios.alpha.CredentialExtension"]
     force_for_new_devices true
-    template_name "Default Web Browser iOS (Dist)"
```

Comment:
template_name is not supported anymore and replaced by “Additional Capabilities” tab for the App ID. We’ve migrated to it for the main iOS app long time ago already.